lib/webworker/WebWorkerMainTemplatePlugin.js

```diff
@@ -45,6 +45,9 @@ class WebWorkerMainTemplatePlugin {
 						"if(!installedChunks[chunkId]) {",
 						Template.indent([
 							"importScripts(" +
+								`"` +
+								mainTemplate.getPublicPath({ hash }) +
```

Comment:
Use `__webpack_require__.p` instead. It's a runtime variable where the publicPath is stored.

Reply:
Updated.